### ISSUE TITLE
TypeError: whisper is not a function

### ISSUE THREAD

**Issue by changyongyong** [ariym/whisper-node#9]
I run my code, but it has a error
this is error
`
const transcript = await whisper(filePath, options);
                         ^

TypeError: whisper is not a function
    at file:///Users/xx/test/test1.js:17:26
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
`

**Comment by deflexable** [comment]
@changyongyong  also having this issue, were you able to fix it?

**Comment by deflexable** [comment]
fixed by changing

```js
import whisper from 'whisper-node';
```

to

```js
import { whisper } from 'whisper-node';
```

**Comment by Juanestban** [comment]
that error can be solved on this pr #20

**Comment by ariym** [comment]
whisper is now the default export so this should work
